Jobs are failing with following errors. Do i need to add a package reference or something inside `System.Utf8String.Experimental.csproj`. How is it suppose to work?

```
F:\workspace\_work\1\s\src\libraries\System.Private.CoreLib\src\System\Numerics\BitOperations.cs(8,33): error CS0234: The type or namespace name 'Arm' does not exist in the namespace 'System.Runtime.Intrinsics' (are you missing an assembly reference?) [F:\workspace\_work\1\s\src\libraries\System.Utf8String.Experimental\src\System.Utf8String.Experimental.csproj]
##[error]src\libraries\System.Private.CoreLib\src\System\Numerics\BitOperations.cs(8,33): error CS0234: The type or namespace name 'Arm' does not exist in the namespace 'System.Runtime.Intrinsics' (are you missing an assembly reference?)
```
@tannergooding , @echesakovMSFT

--- a/src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs
+++ b/src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs
@@ -4,8 +4,8 @@
 
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
-using System.Runtime.Intrinsics.X86;
 using System.Runtime.Intrinsics.Arm;
+using System.Runtime.Intrinsics.X86;
 
 #if SYSTEM_PRIVATE_CORELIB
 using Internal.Runtime.CompilerServices;
@@ -64,7 +64,7 @@ namespace System.Numerics
 
             if (ArmBase.IsSupported)
             {
-                return (int)ArmBase.LeadingZeroCount(value);
+                return ArmBase.LeadingZeroCount(value);
             }
 
             // Unguarded fallback contract is 0->31
@@ -93,7 +93,7 @@ namespace System.Numerics
 
             if (ArmBase.Arm64.IsSupported)
             {
-                return (int)ArmBase.Arm64.LeadingZeroCount(value);
+                return ArmBase.Arm64.LeadingZeroCount(value);
             }
 
             uint hi = (uint)(value >> 32);
@@ -136,7 +136,7 @@ namespace System.Numerics
 
             if (ArmBase.IsSupported)
             {
-                return 31 - (int)ArmBase.LeadingZeroCount(value);
+                return 31 - ArmBase.LeadingZeroCount(value);
             }
 
             // Fallback contract is 0->0
@@ -166,7 +166,7 @@ namespace System.Numerics
 
             if (ArmBase.Arm64.IsSupported)
             {
-                return 63 - (int)ArmBase.Arm64.LeadingZeroCount(value);
+                return 63 - ArmBase.Arm64.LeadingZeroCount(value);
             }
 
             uint hi = (uint)(value >> 32);
@@ -298,7 +298,7 @@ namespace System.Numerics
 
             if (ArmBase.IsSupported)
             {
-                return (int)ArmBase.LeadingZeroCount(ArmBase.ReverseElementBits(value));
+                return ArmBase.LeadingZeroCount(ArmBase.ReverseElementBits(value));
             }
 
             // Unguarded fallback contract is 0->0
@@ -341,7 +341,7 @@ namespace System.Numerics
 
             if (ArmBase.Arm64.IsSupported)
             {
-                return (int)ArmBase.Arm64.LeadingZeroCount(ArmBase.Arm64.ReverseElementBits(value));
+                return ArmBase.Arm64.LeadingZeroCount(ArmBase.Arm64.ReverseElementBits(value));
             }
             uint lo = (uint)value;
 

--- a/src/libraries/System.Utf8String.Experimental/src/System/Runtime/Intrinsics/Intrinsics.Shims.cs
+++ b/src/libraries/System.Utf8String.Experimental/src/System/Runtime/Intrinsics/Intrinsics.Shims.cs
@@ -103,3 +103,19 @@ namespace System.Runtime.Intrinsics.X86
         public static bool TestZ(Vector128<ushort> left, Vector128<ushort> right) => throw new PlatformNotSupportedException();
     }
 }
+
+namespace System.Runtime.Intrinsics.Arm
+{
+    internal abstract class ArmBase
+    {
+        public abstract class Arm64
+        {
+            public const bool IsSupported = false;
+            public static int LeadingZeroCount(ulong value) => throw new PlatformNotSupportedException();
+            public static uint ReverseElementBits(ulong value) => throw new PlatformNotSupportedException();
+        }
+        public const bool IsSupported = false;
+        public static int LeadingZeroCount(uint value) => throw new PlatformNotSupportedException();
+        public static uint ReverseElementBits(uint value) => throw new PlatformNotSupportedException();
+    }
+}

